test/snapshots/examples.ava.js.md

```diff
@@ -6,7 +6,7 @@ Generated by [AVA](https://avajs.dev).
 
 ## svg
 
-> file:///Users/hildjj/track/quence/examples/data.wsd
```

Comment:
oh.  I didn't realize I was leaking local paths like this.  I'll do something to fix this.